lib/new_relic/agent/log_priority.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require 'new_relic/agent/event_aggregator'
+
+# Stateless calculation of priority for a given log event
+module NewRelic
+  module Agent
+    module LogPriority
+      extend self
+
+      SEVERITIES = Logger::Severity.constants.inject({}) do |memo, sev|
+        memo[sev.to_s] = Logger::Severity.const_get(sev)
+        memo
+      end
+
+      TRANSACTION_BONUS = 10
```

Comment:
Do we want to break this out even more with a separate value for errors?

Reply:
Probably deserves its own constant, even if the values in my current scheme are the same :+1: